vk/vk/ViewController/PhotoGalleryViewController.swift

```diff
@@ -15,15 +15,40 @@ final class PhotoGalleryViewController: UIViewController {
 
     // MARK: - Private IBOutlets
 
-    @IBOutlet var photoImageView: UIImageView!
+    @IBOutlet private var photoImageView: UIImageView!
 
     // MARK: - Public Properties
 
-    var imageNames: [String] = []
+    var friend: Friend? {
+        didSet {
+            guard let user = friend else { return }
```

Comment:
```suggestion
            guard let user = friend else { return }
             api.fetchUserPhotos(userID: user.id) { [weak self] photos in
                 self?.friend?.photos = photos
                 self?.friend?.loadProfilePhotos { [weak self] images in
                     self?.images = images
                 }
```
Вынеси в метод у запрос через сервис без родителя